runtime/src/accounts_db.rs

```diff
@@ -729,6 +742,15 @@ mod tests {
         };
     }
 
+    #[macro_export]
+    macro_rules! hashmap {
```

Comment:
Can you Google around for a library here? Maybe this one: https://github.com/bluss/maplit